### ISSUE TITLE
No Network Connection but have network :S

### ISSUE THREAD

**Issue by Pakev** [Karlovsky120/7DaysProfileEditor#38]
on my screen it shows no network connection!

**Comment by Karlovsky120** [comment]
I didn't upload a new release when I dealt with that issue. I'll do it within 24 hours, but if you can't wait, you can try building the source on your own, since the bug is fixed there.

Honestly, I thought A17 was a week away at the time and that I'd just put that up once I update it.

**Comment by Karlovsky120** [comment]
Try version 0.16.3.4 I put up earlier, it should be fine.

**Comment by Pakev** [comment]
Yeah updated versions working but next bug ;) if you have once started it and it creates the config then you cant open it again :S

**Comment by Karlovsky120** [comment]
About the config and starting the program, what happens exactly? I'm curious, but whatever it is I think it shouldn't be a problem to fix.

I'm sure you can open some ttp files, just not the ones you're opening. :D
Can you zip up the blocks and items xmls you're using (they should be in the config folder of the game) along with the ttps that aren't working and upload them somewhere?

I'll see what's going on.

**Comment by Pakev** [comment]
Sure i want to open this file https://cloud.eegcloud.de/index.php/s/aalqI4IBVJ9pc7w

**Comment by Karlovsky120** [comment]
That's useless to me without the block and item xmls.

**Comment by Pakev** [comment]
Sry here you have it all. https://cloud.eegcloud.de/index.php/s/Nk40CRfauDiIsPv

**Comment by Karlovsky120** [comment]
Okay, it seems that the save file version has changed since I last updated the program. A few things in the code are currently a mess and I have no will nor time to deal with it right now. Once A17 is released I'll clean it up and update the save reader part so it handles the new changes.
